utils/args-utils/utils.go

```diff
@@ -0,0 +1,123 @@
+package args_utils
```

Comment:
I'm not sure if you will need additional files related to args-utils, hence I think you should rename this file to args_utils.go and put it in utils package/